### PR TITLE
Add JWKS Signature Validation and Response Signing Conf

### PR DESCRIPTION
**Pull request by isharailanga**
This PR is to add $subject according to the following: 

UK Open Banking JWS Signature Validation- https://docs.google.com/document/d/1wLhkNoc90ZeslA2ZioahBpqti47Ex35A3JE7hnYY8s4/edit?usp=sharing

UK Open Banking Response Signing - https://docs.google.com/document/d/1kDQPTTP6KfhRX65lMV6G82_46470418ltSD3zjeNIf8/edit?usp=sharing

Related to https://github.com/wso2/docs-open-banking/issues/651

### CODE REVIEW

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md [hunk 1]
```diff
@@ -131,7 +131,130 @@ account retrieval. By default, this is disabled and the configuration is set to
...
+           ``` toml
+           [open_banking.uk.jws_signature.obie]
+           org_id="0015800001HQQrZAAX"
+        
+           [open_banking.uk.jws_signature.obie.trusted_anchors]
+           signature_validation="openbanking.org.uk"
+           ```
+
+         5. Configure the API contexts that require signature validation:
+          ``` toml
```
Comment (Ashi1993) [inline en/docs/install-and-setup/configuring-api-manager-for-ob.md:190]: shall we mention about payments API. validation is enabled by default for payments api

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md [hunk 2]
```diff
@@ -131,7 +131,130 @@ account retrieval. By default, this is disabled and the configuration is set to
...
+
+    !!! note
+        This is only available as a WSO2 Update from **WSO2 Open Banking API Manager UK Toolkit Level 3.0.0.x** and
+        **WSO2 Open Banking Identity Server UK Toolkit Level 3.0.0.x** onwards. For more information on updating, see
+        [Getting WSO2 Updates](setting-up-servers.md#getting-wso2-updates).
+
+    ??? tip "Click here to see JWS Response Signing configurations"
+
+         1. Configure the `UKJwsResponseHandlingExecutor` executor. The priority of `UKJwsResponseHandlingExecutor` 
+            must be lower than `ConsentEnforcementExecutor`. For example:
```
Comment (Ashi1993) [inline en/docs/install-and-setup/configuring-api-manager-for-ob.md:205]: this should come before the UKErrorHandlingExecutor

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md [hunk 3]
```diff
@@ -131,7 +131,130 @@ account retrieval. By default, this is disabled and the configuration is set to
...
+        3. Configure the alias and kid values of the signing certificates:
+            - `signing_cert_alias`: The alias of the signing certificate stored in the keystore. Used to sign responses in a production environment. Default value is `wso2carbon`.
+            - `sandbox_signing_cert_alias`: The alias of the signing certificate stored in the keystore. Used to sign responses in a sandbox environment. Default value is `wso2carbon`.
+            - `signing_cert_kid`: The kid value of the corresponding public key of the private key, which is used for signing in a production environment. Default value is `1234`. Mandatory configuration.
+            - `sandbox_signing_cert_kid`: The kid value of the corresponding public key of the private key, which is used for signing in a sandbox environment. Default value is `5678`. Mandatory configuration.
+           ```toml
+           [open_banking.ob_identity_retriever.server]
+           signing_cert_alias="wso2carbon"
+           sandbox_signing_cert_alias="wso2carbon"
+           signing_cert_kid="w7NFeMODzCMOZen_WECAlG9N8gg"
```
Comment (Ashi1993) [inline en/docs/install-and-setup/configuring-api-manager-for-ob.md:232]: shall we add 1234 and 5678 as default configs

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md [hunk 4]
```diff
@@ -131,7 +131,130 @@ account retrieval. By default, this is disabled and the configuration is set to
...
+           add and configure the following:
+           ``` toml
+           [open_banking.uk.signing_config]
+           obie.trusted_anchors.signing = "openbanking.org.uk"
+           obie.org_id="0015800001HQQrZAAX"
+           ```
+        6. Configure the API contexts that require response signing:
+          ``` toml
+          [[open_banking.uk.signing_config.response_sig_required_apis]]
+          api_context=”/open-banking/v3.1/pisp”
```
Comment (Ashi1993) [inline en/docs/install-and-setup/configuring-api-manager-for-ob.md:254]: pisp is added by default so no need to mention again. lets add event api

 api_context=”/open-banking/v3.1/event”

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md (hunk 1 above)
Comment (isharailanga) [reply en/docs/install-and-setup/configuring-api-manager-for-ob.md:191]: Fixed in 0913f39d3e68e6d5c0f106f6490a307c24c16758

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md (hunk 2 above)
Comment (isharailanga) [reply en/docs/install-and-setup/configuring-api-manager-for-ob.md]: Fixed in 0913f39d3e68e6d5c0f106f6490a307c24c16758

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md (hunk 3 above)
Comment (isharailanga) [reply en/docs/install-and-setup/configuring-api-manager-for-ob.md]: Fixed in 0913f39d3e68e6d5c0f106f6490a307c24c16758

File: en/docs/install-and-setup/configuring-api-manager-for-ob.md (hunk 4 above)
Comment (isharailanga) [reply en/docs/install-and-setup/configuring-api-manager-for-ob.md]: Fixed in 0913f39d3e68e6d5c0f106f6490a307c24c16758

**Review by Ashi1993 — APPROVED** [review]

**Pull request merged**